apps/site/src/app/admin/participants/components/ParticipantsTable.tsx

```diff
@@ -50,49 +183,13 @@ function ParticipantsTable({
 
 	return (
 		<Table
-			// TODO: aria labels
-			columnDefinitions={[
-				{
-					id: "uid",
-					header: "UID",
-					cell: (item) => item._id,
-					sortingField: "uid",
-					isRowHeader: true,
-				},
-				{
-					id: "firstName",
-					header: "First name",
-					cell: (item) => item.first_name,
-					sortingField: "firstName",
-				},
-				{
-					id: "lastName",
-					header: "Last name",
-					cell: (item) => item.last_name,
-					sortingField: "lastName",
-				},
-				{
-					id: "role",
-					header: "Role",
-					cell: RoleBadge,
-					sortingField: "role",
-				},
-				{
-					id: "status",
-					header: "Status",
-					cell: ApplicantStatus,
-					sortingField: "status",
-				},
-				{
-					id: "action",
-					header: "Action",
-					cell: ActionCell,
-				},
-			]}
+			{...collectionProps}
 			header={
 				<Header counter={`(${participants.length})`}>Participants</Header>
 			}
-			items={participants}
+			columnDefinitions={columnDefinitions}
+			visibleColumns={preferences.visibleContent}
```

Comment:
Warning: prop is deprecated
> 'visibleColumns' is deprecated.
> deprecated — Replaced by `columnDisplay`.

Reply:
I couldn't get it to work with `columnDisplay`; I think they left collection hooks behind while `Table` props went ahead. This is the best effort implementation for now

Reply:
Got it, works for now then.